src/WebJobs.Script.WebHost/WebHooks/DynamicWebHookReceiverConfig.cs

```diff
@@ -1,38 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNet.WebHooks;
-
-namespace Microsoft.Azure.WebJobs.Script.WebHost.WebHooks
-{
-    public class DynamicWebHookReceiverConfig : IWebHookReceiverConfig
-    {
-        private readonly ISecretManager _secretManager;
-
-        public DynamicWebHookReceiverConfig(ISecretManager secretManager)
```

Comment:
So whats the plan for WebHook support - I see much of it is deleted, and in other places things are commented out.